apps/patient/src/configs/settings.ts

```diff
@@ -35,8 +35,8 @@ export const settings = {
     },
     // Radish Health
     [process.env.REACT_APP_RADISH_HEALTH_ORG_ID as string]: {
-      logo: 'radish_health_logo.svg',
-      accentColor: '#84a98c'
+      logo: 'Red-Radish-Logo.svg',
```

Comment:
can we make - into _ and lower case?
<img width="311" alt="Screen Shot 2023-04-04 at 3 09 46 PM" src="https://user-images.githubusercontent.com/700617/229895645-8ba230f7-d57f-4ff2-8d3e-3d3645a130ff.png">